app/views/admin/tags/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<h1><%= @title = "Tags – #{@tag}" %></h1>
+
+<h2>Taggings</h2>
+
+<%= render partial: 'scope' %>
+<%= render partial: 'filters' %>
+
+<%= link_to 'All', admin_tags_path(model_type: current_klass), class: 'btn' %>
```

Comment:
Think these buttons would be better placed above the filter form, as they're more closely related to the "tag" rather than the list of "taggables" (but appreciate it makes sense that they're under the filter tabs given where they take you).